templates/vue/src/store/mutations.js

```diff
@@ -112,18 +112,29 @@ export function deleteLink(state, { source, target }) {
 }
 
 // quizzes
```

Comment:
Please do a search in the project and identify where we are using the word "quiz" and try to remove or change them as needed.